lib/cropperx.dart

```diff
@@ -226,6 +237,12 @@ class _CropperState extends State<Cropper> {
     );
   }
 
+  void setImageToInitialState() {
+    setState(() {
+      widget.image = widget.initialImage;
```

Comment:
It feels weird updating the widget's image here in the state when the image is not really updated. Wouldn't it be cleaner if we can reset the transformations on the image instead?